packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js

```diff
@@ -75,7 +68,7 @@ Terra.describeViewports('Time Input Twelve Hour Mobile', ['medium'], () => {
       browser.click('#timeInputWithSecond input[name="terra-time-second-time-input-with-second"]');
     });
 
-    Terra.it.isAccessible({ rules: ignoredA11y });
+    Terra.it.isAccessible();
```

Comment:
This could be `Terra.it.validatesElement();`

Reply:
Resolved in 009e0a3